### PR TITLE
Added another method to 2sum Problem

### PR DESCRIPTION
**Pull request by TemitopeAgbaje**
I added another method that can be used to solve 2sum.

kindly review, Thank you

### CODE REVIEW

File: LeetcodeProblems/Algorithms/2Sum.js [hunk 1]
```diff
@@ -28,16 +28,26 @@ Output: [0,1]
...
+  //Another method
+
+  for (let i = 0; i < nums.length; i++) {
+    for (let j = i + 1; j < nums.length; i++) {
+      if (nums[1] + nums[j] === target) {
+        return [i, j];
       }
+    }
   }
 };
```
Comment (ignacio-chiazzo) [inline LeetcodeProblems/Algorithms/2Sum.js:51]: Could you encapsulate the new solution into a new method named `twoSum2`?

There are other examples in other files like `CoinChange`  https://github.com/ignacio-chiazzo/Algorithms-Leetcode-Javascript/blob/08563058f7e93ce1688956cc26d5dc2a98b36d1a/LeetcodeProblems/Algorithms/Coin_Change.js.

Also, could you run the tests for both functions `twoSum` and `twoSum2`?

**Review by ignacio-chiazzo — COMMENTED** [review]
Hey! Thanks for contributing

File: LeetcodeProblems/Algorithms/2Sum.js (hunk 1 above)
Comment (TemitopeAgbaje) [reply LeetcodeProblems/Algorithms/2Sum.js:51]: Alright.. i would fix that

**Comment by TemitopeAgbaje** [comment]
I have updated it.. 
Please review. 
Thank you

**Review by ignacio-chiazzo — APPROVED** [review]
Thank you!

**Pull request merged**